angr/state_plugins/unicorn_engine.py

```diff
@@ -809,6 +926,52 @@ def _map_one_page(self, _uc, addr):
             _UC_NATIVE.activate(self._uc_state, start, length, taint[0] if taint else None)
             return True
 
+    def _get_details_of_blocks_with_symbolic_instrs(self):
+        def _get_register_values(register_values):
+            for register_value in register_values:
+                # Convert the register value in bytes to number of appropriate size and endianness
+                reg_name, reg_size = self.state.arch.vex_reg_offset_to_name[register_value.offset]
+                if self.state.arch.register_endness == 'Iend_LE':
```

Comment:
I would use `archinfo.Endness.LE`, `archinfo.Endness.BE`, and `archinfo.Endness.ME`. Same for line 948.

Reply:
Done.